src/store/local/actions.ts

```diff
@@ -237,7 +238,14 @@ export async function transformMdast(
       }),
     );
   }
-  const data: RendererData = { kind, file, sha256, frontmatter, mdast, references };
+  const data: RendererData = {
+    kind: frontmatter.kernelspec || frontmatter.jupytext ? KINDS.Notebook : kind,
```

Comment:
Is this the correct logic for setting `kind: Notebook` on jupytext md file?

Reply:
This seems correct?! I will take this for a spin today.